### DISCUSSION TITLE
通过OpenAI SDK访问时提示403错误

### DISCUSSION THREAD

**Discussion by Dylan-Li8** [BurnHair/docs#184] [技术支持]
![image](https://github.com/user-attachments/assets/8dba4be6-5450-4e27-b6c8-1c444acff7a0)

**Comment by BennyThink** [comment]
我猜测是 base_url 写错了。有代码吗？

**Comment by Dylan-Li8** [comment]
代码没错，提示是CF的真人验证

Benny ***@***.***> 于2024年8月17日周六 16:08写道：

> 我猜测是 base_url 写错了。有代码吗？
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/BurnHair/discussions/184#discussioncomment-10365021>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A6QUAXQMT3JKYREQGFBKOO3ZR4AGJAVCNFSM6AAAAABMTHCQW2VHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTAMZWGUYDEMI>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
---
Sent with Gmail for Sidekick Browser <https://join.meetsidekick.com/_6ao5y>

**Reply by BennyThink** [reply]
所以我怀疑你的代码写错了啊。我配置的规则非API的请求都会有 managed challenge. 你在OpenAI 的代码里打个断点就知道请求URL是什么了。
